boards/sparkfun/micromod_nrf52840/Kconfig.defconfig

```diff
@@ -0,0 +1,11 @@
+# Sparkfun micromod nrf52840 board configuration
+
+# Copyright (c) 2024 Jerónimo Agulló <jeronimoagullo97@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_MICROMOD_NRF52840
```

Comment:
likewise this becomes `if BOARD_MICROMOD`

Reply:
Ok